File: Data Structures & Algorithms/append-characters-to-string-to-make-subsequence/submission-1.py
class Solution:
    def appendCharacters(self, s: str, t: str) -> int:
        ls = 0
        rs = 0
        n = len(s)
        m = len(t)

        while ls < n and rs < m:
            if s[ls] == t[rs]:
                ls += 1
                rs += 1
            else:
                ls += 1
        
        return m-rs

File: Data Structures & Algorithms/majority-element-ii/submission-1.cpp
class Solution {
public:
    vector<int> majorityElement(vector<int>& nums) {
        unordered_map<int, int> mapp;
        int n = nums.size();
        for(int x: nums){
            mapp[x]++;
        }

        vector<int> res;
        for(auto &[key, value]: mapp){
            if(value > n/3){
                res.push_back(key);
            }
        }
        return res;
    }
};